third_party/silabs/efr32_sdk.gni

```diff
@@ -193,6 +193,7 @@ template("efr32_sdk") {
 
     if (chip_enable_ble_rs911x) {
       _include_dirs += [ "${wifi_sapi_root}/sapi/include" ]
+      _include_dirs += [ "${chip_root}/src/platform/silabs/rs911x" ]
```

Comment:
why is this necessary? the files are already included in the `src/platform/silabs/.../BUILD.gn` files.

Reply:
I believe it is necessary because they have some definitions/ functions declared in `rsi_ble_config.h` and `wfx_sl_ble_init.h` that need to be known by wiseMCU/wiseConnect

There might be a cleaner way to do it tho. would need to investigate

Reply:
Yes, @jmartinez-silabs is correct. and we will move all these files to third-party silabs after refactoring we already discussed this.